tagstudio/src/qt/ts_qt.py

```diff
@@ -189,8 +192,10 @@ def __init__(self, core, args):
         self.settings = QSettings(
             QSettings.IniFormat, QSettings.UserScope, "tagstudio", "TagStudio"
         )
-
-        max_threads = os.cpu_count()
+        if self.settings.value("limit_threads", False, bool):
+            max_threads = self.settings.value("max_threads", os.cpu_count(), int)
+        else:
+            max_threads = os.cpu_count()
```

Comment:
as mentioned previously please make one Pull Request per feature. So this one should be either for the timer, or for the threads count, not both.